src/main.js

```diff
@@ -19,6 +19,8 @@ import Payment from './pages/Payment.vue';
 import Login from './pages/Login.vue';
 import User from './pages/User.vue';
 
+console.log(BUILDDATE + ' ' + VERSION + ' ' + BRANCH);
+
```

Comment:
As it's just an informational message we could use console.info(https://developer.mozilla.org/en-US/docs/Web/API/Console/info) :innocent: